src/binding_utils.rs

```diff
@@ -176,8 +176,30 @@ macro_rules! create_exception {
     };
 }
 
+macro_rules! impl_enum_field {
+    ($pyo3_name: literal, $fn_name: ident, $enum_class: ident, $field_value: expr) => {
```

Comment:
`enum_class` should come first, because you implement for.
And why don't you group in one macro ?

Reply:
I forgot that patterns in rust macros can be repeated. I modified the macro, now a single invocation can generate all fields.
```rs
crate::binding_utils::impl_enum_field!(
    CoreEvent,
    [
        "BACKEND_CONNECTION_CHANGED",
        backend_connection_changed,
        libparsec::core::CoreEvent::BackendConnectionChanged
    ],
   // ...
);
```